drv/transceivers-server/src/main.rs

```diff
@@ -41,7 +44,7 @@ task_slot!(SENSOR, sensor);
 include!(concat!(env!("OUT_DIR"), "/i2c_config.rs"));
 
 #[allow(dead_code)]
-#[derive(Copy, Clone, PartialEq, Eq)]
+#[derive(Copy, Clone, PartialEq, Eq, Count)]
 enum Trace {
     None,
     FrontIOBoardReady(bool),
```

Comment:
since `true` means that the front IO board is ready, and `false` means that it's _not_ ready, maybe we should generate separate counters for those states?
```suggestion
    FrontIOBoardReady(#[count(children)] bool),
```

Reply:
Agreed!